test/distrib/csharp/run_distrib_test.bat

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
I see, that would mean more work, so let's fix the copyrights for .bat files in a separate PR?